File: SDKs/Aspose.Slides_Cloud_SDK_for_.NET/src/Com/Aspose/Slides/Model/ResourceUriElement.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Slides.Model {
  public class ResourceUriElement {
    public ResourceUri Uri { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ResourceUriElement {\n");
      sb.Append("  Uri: ").Append(Uri).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Slides-Cloud-SDK-for-Android/Aspose-Slides-Cloud-SDK-for-Android/src/main/java/com/aspose/slides/model/ResourceUriElement.java
package com.aspose.slides.model;

import com.aspose.slides.model.ResourceUri;
public class ResourceUriElement {
  private ResourceUri Uri = null;
  /**
	 * getUri
	 * Gets ResourceUri
	 * @return Uri
	 */
  public ResourceUri getUri() {
    return Uri;
  }

	/**
	 * setUri
	 * Sets ResourceUri
	 * @param Uri ResourceUri
	 */
  public void setUri(ResourceUri Uri) {
    this.Uri = Uri;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ResourceUriElement {\n");
    sb.append("  Uri: ").append(Uri).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

